### ISSUE TITLE
neuralmagic: Sparse Finetuning 

### ISSUE THREAD

**Issue by irthomasthomas** [irthomasthomas/undecidability#171]
- [ ] [Sparse Finetuning MPT - a neuralmagic Collection](https://huggingface.co/collections/neuralmagic/sparse-finetuning-mpt-65241d875b29204d6d42697d)

    Explore our breakthrough in sparse fine-tuning LLMs! Our novel method maintains downstream accuracy even with >70% sparsity.
